src/mlpack/tests/ann_dist_test.cpp

```diff
@@ -145,29 +141,29 @@ BOOST_AUTO_TEST_CASE(NormalDistributionTest)
   normalDist.LogProbability(x, prob);
 
   // Testing output of log probability for some random mu, sigma and x.
-  BOOST_REQUIRE_CLOSE(prob[0], 1.2586464, 1e-3);
-  BOOST_REQUIRE_CLOSE(prob[1], 0.8751131, 1e-3);
-  BOOST_REQUIRE_CLOSE(prob[2], -0.30579138, 1e-3);
-  BOOST_REQUIRE_CLOSE(prob[3], -5.498411, 1e-3);
+  REQUIRE(prob[0] == Approx( 1.2586464).epsilon(1e-5));
+  REQUIRE(prob[1] == Approx( 0.8751131).epsilon(1e-5));
+  REQUIRE(prob[2] == Approx( -0.30579138).epsilon(1e-5));
+  REQUIRE(prob[3] == Approx( -5.498411).epsilon(1e-5));
```

Comment:
```suggestion
  REQUIRE(prob[0] == Approx(1.2586464).epsilon(1e-5));
  REQUIRE(prob[1] == Approx(0.8751131).epsilon(1e-5));
  REQUIRE(prob[2] == Approx(-0.30579138).epsilon(1e-5));
  REQUIRE(prob[3] == Approx(-5.498411).epsilon(1e-5));
```

Reply:
Hahah it is the space after comman, while copying the numerical value